common/flatpak-dir.c

```diff
@@ -2451,21 +2454,17 @@ flatpak_deploy_data_get_appdata_content_rating (GVariant *deploy_data)
  * Returns: (array length=length zero-terminated=1) (transfer container): an array of constant strings
```

Comment:
`length=length` looks like a copy-paste error, though not the fault of this commit